src/Core/Repositories/SqlServer/CipherRepository.cs

```diff
@@ -226,6 +226,28 @@ public async Task DeleteAsync(IEnumerable<Guid> ids, Guid userId)
             }
         }
 
+        public async Task DeleteByIdsOrganizationIdAsync(IEnumerable<Guid> ids, Guid organizationId)
+        {
+            using (var connection = new SqlConnection(ConnectionString))
+            {
+                var results = await connection.ExecuteAsync(
+                    $"[{Schema}].[Cipher_DeleteByIdsOrganizationId]",
+                    new { Ids = ids.ToGuidIdArrayTVP(), organizationId = organizationId },
```

Comment:
Generally, I name the params with PascalCasing since that is what they are in SQL. I suppose it will map fine still.

Reply:
Fixed this